polyfills/String/prototype/normalize/detect.js

```diff
@@ -0,0 +1 @@
+'normalize' in String.prototype && "".normalize.length === 0
```

Comment:
Does it need to be a string literal or could we check on the String.prototype directly?
```suggestion
'normalize' in String.prototype && String.prototype.normalize.length === 0
```